src/app/pages/charging-stations/charging-plans/charging-plans-list-table-data-source.ts

```diff
@@ -51,6 +53,19 @@ export class ChargingPlansListTableDataSource extends TableDataSource<ChargingPr
       this.setStaticFilters([{ WithChargingStation: 'true' }]);
     }
     this.initDataSource();
+    this.initFilters();
+  }
+
+  public initFilters() {
+        // Charging Station
+        const chargingStationID = this.windowService.getSearch('ChargingStationID');
```

Comment:
As discussed, use the transaction id to jump to this screen using the Search field.